index.html

```diff
@@ -2029,6 +1996,13 @@ <h2>Verification Relationships</h2>
 <code><a>keyAgreement</a></code> property needs to be used for that.
       </p>
       <p>
+The <a>DID document</a> does not express revoked keys using a <a>verification
```

Comment:
Noting this moved again, but no objection on my behalf here. With that in mind, my concerns metioned in https://github.com/w3c/did-core/pull/636#pullrequestreview-590088398 remain.